docs_src/microservices/security/edgex-secrets-config-proxy.md

```diff
@@ -0,0 +1,142 @@
+% edgex-secrets-config-proxy(1) User Manuals edgex-secrets-config-proxy(1)
+
+# NAME
+
+**edgex-secrets-config-proxy** – Configure EdgeX proxy service
+
+# SYNOPSIS
+
+**edgex-config** proxy [OPTIONS] SUBCOMMAND [ARG...]
+
+# DESCRIPTION
+
+**edgex-secrets-config-proxy** configures the EdgeX proxy service.
+
+This command is used to configure the TLS certificate for external connections, create authentication tokens for inbound proxy access, and relevant utility functions.
+
+These commands require access to the secret store master key in order to generate temporary secret store access credentials.
+
+# OPTIONS
+
+  * **-m** _/path/to/resp-init.json_ (optional)
+
+    Points to location of secret store master key. Defaults to /vault/config/assets/resp-init.json.
+
+# COMMANDS
+
+  * **tls**
+
+    Configure inbound TLS certificate. This command will configure the TLS secrets into the secret store and re-deploy them to Kong. Requires additional arguments:
+
+    * --certificate _/path/to/certchain.pem_ (required)
+
+      Path to TLS leaf certificate (with optional appended certificate chain, not including a root certificate).
+
+    * --key _/path/to/private.pem_ (required)
+
+      Path to TLS private key.
+
+  * **adduser**
+
+    Create a proxy user using JWT public/private key authentication. Requires additional arguments:
+
+    * --oauth (unimplemented; reserved for future use)
+
+      Create user using OAuth authentication plugin.
+
+    * --jwt (required)
+
+      Create user using JWT authentication plugin.
+
+    * --algorithm HS256 | ES256 (required for JWT method)
+
+      Algorithm used for signing the JWT.
+
+    * --public\_key _public\_key.pem_ (required for JWT method)
+
+      Public key used to validate the JWT. This key is assumed to have been pre-created using some external mechanism such as a TPM, HSM, openssl, or other method.
```

Comment:
The use of **.pem** implies that this is a certificate with an embedded public key...

Reply:
I disagree.  .pem is pretty generic and can mean a pem-encoded anything.